cilium-cli/status/status.go

```diff
@@ -433,11 +441,44 @@ func (s *Status) Format() string {
 		}
 	}
 
+	header = "Taints:"
+	for taint, nodes := range getCiliumNodeTaints(s.NodeAnnotations) {
+		fmt.Fprintf(w, "%s\t%s\t%s\n", header, defaults.AgentDaemonSetName, trimNodeNames(nodes))
+		for _, line := range strings.Split(taint, "\n") {
+			fmt.Fprintf(w, "\t\t%s\n", line)
+		}
+		header = ""
+	}
```

Comment:
Would be nice to display these sorted alphabetically, like e.g. errors and warnings above. Otherwise these lines will have a different sorting order each time `cilium status` is invoked or during `cilium status --wait` which makes it a bit hard to follow what's going on. Also see https://github.com/cilium/cilium/pull/34927 for context.

Reply:
Ah good catch, thanks!